### PR TITLE
Elgato.CameraHub version 1.4.0.770

### PR DESCRIPTION
**Pull request by sanyer**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/86112)

### CODE REVIEW

File: manifests/e/Elgato/CameraHub/1.4.0.770/Elgato.CameraHub.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://edge.elgato.com/egc/windows/echw/1.4/CameraHub_1.4.0.770_x64.msi
+  InstallerSha256: 9EA79120873BBE8EA6749247477CA375F03A8D76E44E8DAFB8F79D0F417A44DB
+  ProductCode: '{75FD4F4E-9170-4792-8C8C-5256A9C13E0C}'
+  AppsAndFeaturesEntries:
+  - DisplayVersion: 1.3.0.595
```
Comment (Trenly) [inline manifests/e/Elgato/CameraHub/1.4.0.770/Elgato.CameraHub.installer.yaml:22]:
```suggestion
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by sanyer** [comment]
@Trenly please advise, I don't see anything meaningful in logs on Azure DevOps 🤷🏻‍♂️

**Comment by Trenly** [comment]
> @Trenly please advise, I don't see anything meaningful in logs on Azure DevOps 🤷🏻‍♂️

Looks like Elgato has blocked the validation pipelines -
```
2022-10-24 15:06:51.268 [CLI ] Caught wil::ResultException: D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(62)\WindowsPackageManager.dll!00007FFF7F77C156: (caller: 00007FFF7F77C5B4) Exception(4) tid(f38) 80190193 Forbidden (403).

    Msg:[Download request status is not success.] 
```

**Comment by ItzLevvie** [comment]
Close with reason: Duplicate of PR 85060;

**Pull request closed without merging**